static/js/settings_bots.js

```diff
@@ -471,6 +471,20 @@ exports.set_up = function () {
         $(this).attr("href", exports.generate_zuliprc_uri(bot_id));
     });
 
+    $("#active_bots_list").on("click", "button.copy_zuliprc", function () {
+        var bot_info = $(this).closest(".bot-information-box");
+        var email = bot_info.find(".email .value").text();
+        var api_key = bot_info.find(".api_key .api-key-value-and-button .value").text();
+        var data = exports.generate_zuliprc_content($.trim(email), $.trim(api_key));
+        // Generates a disposable text area and deletes it after done copying.
+        var element = document.createElement('textarea');
+        element.append(document.createTextNode(data));
+        document.body.append(element);
+        element.select();
+        document.execCommand('copy');
+        element.parentNode.removeChild(element);
+    });
```

Comment:
Why don't we use `ClipboardJS` here, https://clipboardjs.com/#usage, see how we did this for "Copy link to conversation"?